src/erc20/contracts.ts

```diff
@@ -4,37 +4,58 @@ import networkIds from "../networks";
 
 const contracts = {
   abi: ERC20Abi,
+  eth: {
+    symbol: 'ETH',
+    decimals: 18,
+    address: {
+      [networkIds.mainnet]: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
```

Comment:
Is this a universal thing? Or is it just an AAVE thing?

Reply:
Its almost a universal thing on all on-chain DEX's see: https://github.com/CryptoManiacsZone/1split/blob/master/OneSplit.full.sol